docs/common-coreboot-docs/dumping_logs.md

```diff
@@ -0,0 +1,58 @@
+# Dumping logs
+
+When facing an issue on a unique hardware configuration on the end user side it
+may prove useful to submit system logs to Dasharo team for diagnosis and
+possibly problem solution. This section describes how to dump various logs from
+a running system.
+
+## System information
+
+One may use [Dasharo Tools Suite HCL report](/common-coreboot-docs/dasharo_tools_suite/#dasharo-hcl-report)
+or [fwdump-docker image](https://github.com/3mdeb/fwdump-docker) to
```

Comment:
@miczyg1 fwdupm should not be used, why we have it here?

Reply:
One reason is this https://github.com/Dasharo/meta-dts/issues/21